docs/schema/default-workflow-conditions-workflowinfo.md

```diff
@@ -47,7 +48,7 @@ The following table lists the default workflow conditions that are included with
 </thead>
 <tbody>
 <tr class="odd">
-<td align="left"><p>**Compare** *<list or document library name>* **field**</p></td>
+<td align="left"><p>**Compare** *\&ltlist\&gt* **field**</p></td>
```

Comment:
Let's keep the original formatting the same with your HTML solution, so that line 51 reads as:
`<td align="left"><p>**Compare** *&ltlist or document library name&gt* **field**</p></td>`

Reply:
Changed as requested.